client/src/main/java/org/apache/rocketmq/client/Validators.java

```diff
@@ -33,8 +33,12 @@
  * Common Validator
  */
 public class Validators {
-    public static final int CHARACTER_MAX_LENGTH = 255;
-    public static final int TOPIC_MAX_LENGTH = 127;
+    public static final int DEFAULT_TOPIC_MAX_LENGTH = 192;
+    public static final int DEFAULT_CONSUMER_GROUP_MAX_LENGTH = 255;
+    // not thread safe
+    public static int consumerGroupMaxLength = DEFAULT_CONSUMER_GROUP_MAX_LENGTH;
```

Comment:
If I actually set this length to 255, the retry topic may be limited by the system topic length. we need reduce the length to safe water.

Reply:
It seems that  Validators.checkTopic() method only used in MQProducer. it does not harm sendmessageback logic in consume case.